CrossrefReferenceLinkingSettingsForm.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+/**
+ * @file CrossrefReferenceLinkingSettingsForm.inc.php
+ *
+ * Copyright (c) 2014-2023 Simon Fraser University
+ * Copyright (c) 2003-2023 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file LICENSE.
+ *
+ * @class CrossrefReferenceLinkingSettingsForm
+ * @ingroup plugins_generic_crossrefReferenceLinking
+ *
+ * @brief Form for journal managers to setup the reference linking plugin
+ */
+
+namespace APP\plugins\generic\crossrefReferenceLinking;
+
+use APP\core\Application;
+use APP\template\TemplateManager;
+use PKP\form\Form;
+use PKP\form\validation\FormValidatorCSRF;
+use PKP\form\validation\FormValidatorPost;
+use PKP\linkAction\LinkAction;
+use PKP\linkAction\request\RedirectAction;
+
+
+class CrossrefReferenceLinkingSettingsForm extends Form
+{
+
+	public int $contextId;
+
+	public CrossrefReferenceLinkingPlugin $plugin;
+
+	public function __construct(CrossrefReferenceLinkingPlugin $plugin, int $contextId)
```

Comment:
You can use a new syntax here:

```php
public function __construct(public CrossrefReferenceLinkingPlugin $plugin, public int $contextId) {
...
}
```

Then drop the property declarations above and the attributions below.